src/kernelsearch/search.py

```diff
@@ -36,20 +36,63 @@ def _get_duration_lims(period,
                        M_star_min,
                        M_star_max):
 
+    #max_duration = grid.T14(
+    #    R_s=R_star_max,
+    #    M_s=M_star_min,
+    #    P=period,
+    #    small=False  # large planet for long transit duration
+    #)
+    #min_duration = grid.T14(
+    #    R_s=R_star_min,
+    #    M_s=M_star_max,
+    #    P=period,
+    #    small=True  # small planet for short transit duration
+    #)
+
+    rho_min = 1.41 * M_star_min / (R_star_max ** 3)
```

Comment:
Could you add a comment explaining what is being done here? Explain that 1.41 is the solar density in g/cm^3.